### PR TITLE
created tests for n, nn, sw, and like

### PR DESCRIPTION
**Pull request by rmbasilio**
Tests for n, nn, sw, and like

### CODE REVIEW

File: src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java [hunk 1]
```diff
@@ -116,4 +116,95 @@ public void testPagination01() throws Exception
...
+   public void testN01() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+      ObjectNode json = null;
+
+      res = service.get(url("orders?limit=5&n(shipRegion)")).statusOk();
+      json = res.getJson();
+      assertEquals(5, json.find("meta.pageSize"));
+      assertEquals(5, json.getArray("data").length());
```
Comment (wellsb1) [inline src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java]: This needs to actually check that the shipregion field is null.

See comments on https://github.com/RocketPartners/rocket-inversion/pull/97 for consistency.

File: src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java [hunk 2]
```diff
@@ -116,4 +116,95 @@ public void testPagination01() throws Exception
...
+   public void testN02() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+      ObjectNode json = null;
+
+      res = service.get(url("orders?limit=5&n(shipCountry)")).statusOk();
+      json = res.getJson();
+      assertEquals(0, json.find("meta.pageSize"));
+      assertEquals(0, json.getArray("data").length());
```
Comment (wellsb1) [inline src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java]: same, need to check the data in addition to results size.

**Review by wellsb1 — CHANGES_REQUESTED** [review]
Every one of theses test cases needs to check the actual data not just the results length.

Please see comments on @hayas PR #97 for consistency.

Also, if these operators are not tested in TestSqlQuery, please add tests for them there too.

**Comment by rmbasilio** [comment]
> Every one of theses test cases needs to check the actual data not just the results length.
> 
> Please see comments on @hayas PR #97 for consistency.
> 
> Also, if these operators are not tested in TestSqlQuery, please add tests for them there too.

Changes made to match updated requirements. I believe I saw these operators in TestSqlQuery, though it's a little bit hard to tell for sure with all the tests cases jammed into a single test.

**Review by wellsb1 — APPROVED** [review]

**Pull request merged**